### ISSUE TITLE
Update to material-ui 5.x

### ISSUE THREAD

**Issue by JL102** [mediamonkeyserver/mms#61]
Material UI renamed from 4.x to 5.x, so we will need to change out dependencies from:
```json
    "@material-ui/core": "^4.9.9",
    "@material-ui/icons": "^4.0.1",
```
to:
```json
    "@mui/material": "^5.3.0",
    "@mui/icons-material": "^5.3.0",
```
We cannot update from react 16.x to 17.x until updating to MUI 5.x.

Additionally, [material-ui-search-bar](https://github.com/TeamWertarbyte/material-ui-search-bar/blob/master/package.json) has not been updated to MUI 5.x, so we'd need to fork it or find an appropriate replacement.

**Comment by jirihajek** [comment]
Right, but since MUI 5.x isn't compatible to 4.x, we won't rush it. That said, from what I tested, 5.x seems to go in the right direction, so we'll upgrade sometimes soon.
